Go/Go.sublime-syntax

```diff
@@ -416,45 +432,69 @@ contexts:
 
   match-floats:
     # Integer, no fraction, exponent
-    - match: \d+({{exponent}})\d+
-      scope: constant.numeric.float.go
-      captures:
-        1: punctuation.separator.exponent.go
+    - match: '{{ddigits}}{{exponent}}{{ddigits}}'
+      scope: constant.numeric.float.decimal.go
     # Integer, fraction, optional exponent
-    - match: \d+(\.)\d+(?:({{exponent}})\d+)?
-      scope: constant.numeric.float.go
+    - match: '{{ddigits}}(\.){{ddigits}}(?:{{exponent}}{{ddigits}})?'
+      scope: constant.numeric.float.decimal.go
       captures:
         1: punctuation.separator.decimal.go
-        2: punctuation.separator.exponent.go
     # Dot without fraction
     - match: \d+\.(?:{{exponent}}\d+)?
       scope: invalid.deprecated.go
     # Dot without integer
     - match: \.\d+(?:{{exponent}}\d+)?
       scope: invalid.deprecated.go
+    # Hex float literal, no fraction
+    - match: '(0[xX]){{hdigits}}[pP][+-]?{{ddigits}}(i)?'
```

Comment:
Just see the `(i)?` here. With regards to PR #2135 we should propably scope hexadecimal imaginary numbers as `constant.numeric.imaginary.hexadecimal` in Go as well?

Reply:
Resolved in ea7018d. The same was true for hex, binary, and octal integers — so I updated those as well. Thanks.